features/theme/index.tsx

```diff
@@ -36,6 +36,16 @@ export interface CreateThemeOptions {
     componentDefaults?: Record<ModifiableComponent, Record<string, any>>;
 }
 
+// Utility function to handle color scheme setting logic
```

Comment:
I suggest moving the comment right above the `if statement` or removing it since the function is self-explanatory